.github/workflows/docker.yml

```diff
@@ -19,27 +19,31 @@ jobs:
         ATB_NEXT_PUBLIC_FIREBASE_AUTH_DOMAIN=${{ github.event_name == 'release' && 'atb-mobility-platform.firebaseapp.com' || 'atb-mobility-platform-staging.firebaseapp.com' }}
         ATB_NEXT_PUBLIC_FIREBASE_APP_ID=${{ github.event_name == 'release' && '1:827196677776:web:c4b1d8924bb5778bf50215' || '1:939812594010:web:0308b2a9cdc80b0d069363' }}
         ATB_NEXT_PUBLIC_FIREBASE_MEASUREMENT_ID=${{ github.event_name == 'release' && '' || '' }}
+        ATB_NEXT_PUBLIC_IN_STAGING=${{ github.event_name == 'atb-mobility-platform-staging'}}
```

Comment:
I think it would be better if this env variable said which environment it is, rather than a boolean for being staging or not. Something like `'dev' | 'staging' | 'prod'`. 

Also, the environment should be the same for everyone, so do we need to prefix these and have multiple ones?

Reply:
I think this should do the trick

`NEXT_PUBLIC_ENVIRONMENT=${{ github.event_name == 'release' && 'prod' || 'staging' }}`

Reply:
I´ll update the logic.